wazo_confd/plugins/extension_feature/api.yml

```diff
@@ -17,15 +17,15 @@ paths:
           description: Extension feature list
           schema:
             $ref: '#/definitions/ExtensionFeatureItems'
-  /extensions/features/{extension_id}:
+  /extensions/features/{extension_uuid}:
```

Comment:
That's an API change. Does portal or wazo-ui use `/extensions/features/<id>`?

Reply:
Internal ticket created : https://wazo-dev.atlassian.net/browse/NES-2163
And new PR created too: https://github.com/wazo-platform/wazo-ui/pull/149